pyproject.toml

```diff
@@ -47,7 +47,7 @@ classifiers = [
     "Topic :: Software Development :: Quality Assurance",
 ]
 dependencies = [
-    "flake8 >= 3.2.1, < 7.0.0",
+    "flake8 >= 5.0.4, < 7.0.0",
```

Comment:
Note that we test running with flake8 <6 in CI, as it's impossible to install flake8 >=6 on Python 3.7, and we test with Python 3.7 in CI, so the Python 3.7 CI job tests running with flake8 <6.